infra.bs

```diff
@@ -55,14 +55,12 @@ urlPrefix: https://tc39.github.io/ecma262/#; spec: ECMA-262;
 
 <h2 id=usage>Usage</h2>
 
-<p>To make use of the Infra Standard in a document titled <var>X</var>, use
-<samp><var>X</var> depends on the Infra Standard</samp>. Additionally, cross-referencing terminology
-is encouraged to avoid ambiguity.
+<p>To make use of this standard in a document titled <var>X</var>, use:
 
-<p>Specification authors are also encouraged to add their specification to the
-<a href=https://github.com/whatwg/infra/wiki/Dependent-specs>list of dependent specifications</a> in
-order to help the editors ensure that any future breaking changes to the Infra Standard are
-correctly reflected by any such dependencies.
+<p><samp><var>X</var> depends on <cite>Infra</cite>. [[!Infra]]</samp>
+<!--                                                 Yo Dawg -->
```

Comment:
I don't get it :(

Reply:
Infra referencing Infra seemed close enough to https://knowyourmeme.com/memes/xzibit-yo-dawg.